app/controllers/recipes_controller.rb

```diff
@@ -5,11 +5,28 @@ def index
     @recipes = @user.recipes.all
   end
 
+  def new
+    @recipe = Recipe.new
+  end
+
   def show
     @recipe = Recipe.includes(:recipe_foods).find(params[:id])
+    @recipe_foods = @recipe.recipe_foods
     @foods = Food.all
   end
 
+  def create
+    @user = User.find(params[:user_id])
+    @recipe = @user.recipes.new(recipe_params)
+    if @recipe.save
+      flash[:notice] = 'Recipe was successfully created'
+      respond_to? user_recipes_path(user_id: params[:user_id])
+    else
+      flash[:alert] = "Couldn't create the recipe!"
+      render :new
+    end
+  end
+
```

Comment:
Nice. This could cause conflict with a previous PR. Please review carefully in the in the merge.